python/lsst/cell_coadds/_identifiers.py

```diff
@@ -167,6 +167,7 @@ def from_data_id(cls, data_id: DataCoordinate) -> ObservationIdentifiers:
         # a fully-expanded data ID, but this isn't embedded in the types.
         return cls(
             instrument=data_id["instrument"],  # type: ignore
+            # TODO: Replace the deprecated pack method in DM-40635
```

Comment:
I have this ticket assigned to myself, but I think it'd be easier if you take this up Jim. I'd happy to review it, of course.

Reply:
:+1: , please go ahead and assign it to me.